doc/source/advanced.md

```diff
@@ -75,6 +75,59 @@ and **google cloud's ingress controller**.
 This should provision a certificate, and keep renewing it whenever it gets close
 to expiry!
 
+
+## Using PostgreSQL as database for you Hub.
+
+Four types of settings are allowed for storing hub state, as described in
+the key ``hub.db.type``:
+
+- ``sqlite-pvc``
+- ``sqlite-memory``
+- ``mysql``
+- ``postgres``
+
+Using PostgreSql or MySQL provides the main advantage to allow seemless update
```

Comment:
Can you explain a bit more what you mean by this paragraph?  What differences are you seeing between sqlite / postgres? When the hub pod restarts, here are the following sequence of actions:

1. Running hub pod pod dies
2. New hub pod is started and scheduled
3. (if using sqlite on a PVC) If new pod is on a different node than old pod (which we've taken steps to avoid by using pod affinity rules), disk must detach from old node and attach to new node
4. hub process starts, and checks all user servers to see if they are running (this takes the longest time)
5. It resumes service

Using an external database will make step (3) go faster, but (4) is the biggest cause for concern.

Reply:
1. with sqlite, "Running hub pod pod dies" yes. With postgres, i clearly see it waits for the new pod to be up before being terminated.
2. ok
3. ok (i guess the termination of the old hub occurs here)
4. ok (i guess there is also a potential DB scheme migration can occurs here, so the service might be unavailable here)
5. ok

i'll check tomorow more carefully, but from my experiment and reading the hub deployment, we see there is a rolling update strategy applied

Reply:
Indeed, a rollingupdate strategy is applied when using postgres! I've reverted that in https://github.com/jupyterhub/zero-to-jupyterhub-k8s/pull/576 - I thought I already had, but had forgotten! Apologies for the confusion!